selfdrive/ui/qt/sidebar.cc

```diff
@@ -101,11 +101,15 @@ void Sidebar::paintEvent(QPaintEvent *event) {
   p.drawImage(60, 1080 - 180 - 40, home_img);
 
   // network
-  p.drawImage(58, 196, signal_imgs[strength]);
+  p.drawImage(60, 190, signal_imgs[strength]);
   configFont(p, "Open Sans", 35, "Regular");
   p.setPen(QColor(0xff, 0xff, 0xff));
-  const QRect r = QRect(50, 247, 100, 50);
+  const QRect r = QRect(30, 220, 240, 50);
   p.drawText(r, Qt::AlignCenter, network_type[net_type]);
+  // Future line for current IP Address
+  //configFont(p, "Open Sans", 28, "Regular");
+  //const QRect s = QRect(30, 260, 240, 50);
+  //p.drawText(s, Qt::AlignCenter, "000.000.000.000");
```

Comment:
Arnepilot has the IP address sent from python.  https://github.com/arne182/ArnePilot/blob/8ed7b03f43d6def9179c862f657498ccaffa5329/selfdrive/ui/sidebar.cc#L67

I'm not sure there is going to be an easy way to get this.  From what I can tell, you'll have to run a shell command to dump the IP address, capture/parse the output and then send it to the UI using the pub/sub messages.